shiny/SDAdashboard/www/style.css

```diff
@@ -425,10 +437,18 @@ input[type=submit]:hover {
   .card-group > .card:not(:first-child):not(:last-child):not(:only-child) {
     border-radius: 0;
   }
-  .card-group > .card:not(:first-child):not(:last-child):not(:only-child) .card-img-top,
-  .card-group > .card:not(:first-child):not(:last-child):not(:only-child) .card-img-bottom,
-  .card-group > .card:not(:first-child):not(:last-child):not(:only-child) .card-header,
-  .card-group > .card:not(:first-child):not(:last-child):not(:only-child) .card-footer {
+  .card-group
```

Comment:
not sure if this is still parsed correctly by CSS after this change (not really familiar with what happens here).

Reply:
No, It will not break it.